tests/components/local_calendar/test_config_flow.py

```diff
@@ -14,6 +22,38 @@
 from tests.common import MockConfigEntry
 
 
+@pytest.fixture
+def mock_process_uploaded_file(tmp_path: Path) -> Generator[MagicMock, None, None]:
```

Comment:
Very nice test! Can you also exercise the case where the ICS content is invalid and fails to parse?